src/plugins/base.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, bot):
         """
         self.bot = bot
 
-    def handle_tokens(self, channel, msg, keywords, callback):
```

Comment:
what exactly do we gain by moving channel to args? If you need args just add args, but I don't think moving channel is a good idea, because most of our bot functions accept channel as the first parameter (part_user, change_user, add_user, ...), why should handle_tokens be special?